jp2_online/templates/captura/create_integrante_form.html

```diff
@@ -0,0 +1,4 @@
+<form action="{% url 'captura:create_integrante' familia.id %}" method="post">
```

Comment:
I don't understand why is this a form instead of a link to captura:create_integrante ?